templates/room.html

```diff
@@ -907,19 +967,88 @@
       orderList.splice(orderList.indexOf(target),1);
       return orderList;
     }
+    
+    
+    //returns boolean that tells you if it's your turn or not.
+    function isMyTurn(){
+        return currentTurn.turnUser === username;
+    }
+    
+    //increments turn class. Should be used after a move.
+    function nextTurn(){
```

Comment:
this logic seems okay, but i will use Mod like round% total_num_of_players which will be easier i think.